Solutions/Proofpoint On demand(POD) Email Security/Data Connectors/ProofPointEmailSecurity_CCP/ProofpointPOD_PollingConfig.json

```diff
@@ -0,0 +1,92 @@
+[
+	{
+		"type": "Microsoft.SecurityInsights/dataConnectors",
+		"apiVersion": "2021-10-01-preview",
+		"name": "ProofpointPodMessageConnection",
+		"location": "{{location}}",
+		"kind": "WebSocket",
+		"properties": {
+			"connectorDefinitionName": "ProofpointCCPDefinition",
+			"dataType": "ProofpointPODMessage_CL",
+			"auth": {
+				"type": "APIKey",
+				"ApiKey": "{{apiKey}}",
+				"apiKeyName": "Authorization",
+				"apiKeyIdentifier": "Bearer",
+				"type": "APIKey"
+			},
+			"request": {
+				"httpMethod": "Get",
+				"apiEndpoint": "wss://logstream.proofpoint.com:443/v1/stream",
+				"startTimeAttributeName": "sinceTime",
+				"endTimeAttributeName": "toTime",
+				"queryTimeFormat": "yyyy-MM-ddTHH:mm-0000",
+				"queryParameters": {
+					"cid": "{{clusterId}}",
+					"type": "message"
+				},
+				"headers": {},
+				"rateLimitQPS": 20,
+				"queryWindowInMin": 5,
+				"retryCount": 2,
+				"timeoutInSeconds": 180,
+				"logResponseContent": true
+			},
+			"response": {
+				"eventsJsonPaths": [
+					"$"
+				]
+			},
+			"dcrConfig": {
+				"streamName": "Custom-ProofpointPodMessage",
+                "dataCollectionEndpoint": "{{dataCollectionEndpoint}}",
+                "dataCollectionRuleImmutableId": "{{dataCollectionRuleImmutableId}}"
+			}
+		}
+	},
+	{
+		"type": "Microsoft.SecurityInsights/dataConnectors",
+		"apiVersion": "2021-10-01-preview",
+		"name": "ProofpointPodMailLogConnection",
+		"location": "{{location}}",
+		"kind": "WebSocket",
+		"properties": {
+			"connectorDefinitionName": "ProofpointCCPDefinition",
+			"dataType": "ProofpointPODMailLog_CL",
+			"auth": {
+				"type": "APIKey",
+				"ApiKey": "{{apiKey}}",
+				"apiKeyName": "Authorization",
+				"apiKeyIdentifier": "Bearer",
+				"type": "APIKey"
+			},
+			"request": {
+				"httpMethod": "Get",
+				"apiEndpoint": "wss://logstream.proofpoint.com:443/v1/stream",
+				"startTimeAttributeName": "sinceTime",
+				"endTimeAttributeName": "toTime",
+				"queryTimeFormat": "yyyy-MM-ddTHH:mm-0000",
+				"queryParameters": {
+					"cid": "[[parameters('clusterId')]",
+					"type": "maillog"
+				},
+				"headers": {},
+				"rateLimitQPS": 20,
+				"queryWindowInMin": 5,
+				"retryCount": 2,
+				"timeoutInSeconds": 180,
+				"logResponseContent": true
```

Comment:
Do we need "logResponseContent": true as this may be increase the load and latency.